trite-osprey/src/App.css

```diff
@@ -0,0 +1,156 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 10s linear;
+  height: 80px;
+  filter: invert(100%);
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-intro {
+  font-size: large;
+  padding: 1em;
+}
+h2{
+  font-family: 'Permanent Marker', cursive;
+  font-size: 3em;
+
+}
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+.listclass {
+  display: flex;
+  align-items: center;
+  z-index: 1;
+}
+
+input[type = text]{
+  text-align: center;
+  padding: 2px 2px;
+  box-shadow: 0 1px #ddd;
+  width:384px;
+  outline: none;
+  display:block;
+  color:#222;
+  margin: 0 auto 20px;
+  margin-top: 20px;
+  font-family: 'Permanent Marker', cursive;
+  font-weight: lighter;
+}
+
+.center_section{
```

Comment:
you were using dashes and now you're using underscores as word delimiters. Stay consistent.